houston/command.py

```diff
@@ -222,6 +222,11 @@ def __getitem__(self, name: str) -> Any:
             raise KeyError(msg)
         return getattr(self, param._field)
 
+    def __hash__(self) -> int:
+        all_params = [self[p.name] for p in self.__class__.parameters]
+        all_params.insert(0, self.uid)
```

Comment:
Maybe `(self.uid,) + (self[p.name] for p in self.__class__.parameters)`?

Reply:
`(self[p.name] for p in self.__class__.parameters)` gives me a `generator` but `tuple(self[p.name] for p in self.__class__.parameters)` should be fine.

Reply:
Ah, that makes sense. Sure.